docs/developer/payments/lifecycle.mdx

```diff
@@ -78,6 +78,34 @@ Charge status depends on:
 
 The difference between `total` and `totalGrantedRefund` is compared with the sum of **charged** and **authorized** transactions. The result is one of the statuses: `PARTIAL`, `FULL`. If no transaction has been charged, the status is `NONE`.
 
+#### Authorization status calculation for Checkout
+
+The status is calculated based on all transactions assigned to the `Checkout`. 
```

Comment:
suggestion: I think we should use object names, to avoid confusion between Transaction and TransactionItem (the first one is related to old Payments API)

```suggestion
The status is calculated based on all `TransactionItem`'s assigned to the `Checkout`. 
```